assets/scss/_calendar.scss

```diff
@@ -186,6 +186,10 @@ ul.calendar {
   }
 }
 
+ul.calendar-clear {
```

Comment:
added as latest script policy means that direct styles in files are disallowed without a nonce, so easier to define here